File: Conjuntos/js/alunos.js
function studentsCount(courses) {
    let set = new Set();

    for(let i = 0; i < courses.length; i++) {
        for (let j = 0; j < courses[i].length; j++) {
            set.add(courses[i][j]);
        }
    }

    return set.size;
}

console.log(studentsCount([
  [15, 21, 80, 42],
  [21, 80, 47],
  [12, 21, 47, 35]
]
))

File: Conjuntos/python/alunos.py
def students_count(courses: list[list[int]]):
    idSet = set()
    for c in courses:
        for s in c:
            idSet.add(s)
    
    return len(idSet)


print(students_count([
  [15, 21, 80, 42],
  [21, 80, 47],
  [12, 21, 47, 35]
]
))
